Reword a complex sentence on EF page (#4370)
ethereum/ethereum-org-website: src/content/foundation/index.md

The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.

> The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor they are the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).

Reviewer: ```suggestion
The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, and they aren't the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
```

Author: yes thanks !
